scripts/meson.build

```diff
@@ -16,5 +15,5 @@ foreach s: builtin_scripts
     builtin_lua_scripts += custom_target('Converted ' + s + '.bin',
                                          input: compiled_script,
                                          output: s + '.bin' + '.h',
-                                         command: [xxd, '-i', '@INPUT@', '@OUTPUT@'])
+                                         command: [files('xxdi.py'), '@INPUT@', '@OUTPUT@'])
```

Comment:
Could you move this script to a `tools/` subdirectory in the root? That's the mpv and ffmpeg way, and I'd prefer to keep `scripts/` and `DOCS/examples/` for the project's builtin scripts and example scripts respectively.

Reply:
Done in a separated commit as it requires changing C files.